cmd/goProbe/goProbe.go

```diff
@@ -118,19 +119,17 @@ func main() {
 
 	// configure api server
 	var (
-		apiServer  *server.Server
+		apiServer  *gpserver.Server
 		apiOptions = []server.Option{
-			server.WithDBPath(config.DB.Path),
 			// Set the release mode of GIN depending on the log level
 			server.WithDebugMode(
 				logging.LevelFromString(config.Logging.Level) == logging.LevelDebug,
 			),
+			server.WithProfiling(config.API.Profiling),
+			server.WithMetrics(config.API.Metrics),
 		}
 	)
 
-	// if config.API.Metrics {
```

Comment:
Deadcode?

Reply:
More a reminder to bring this back eventually.